hbw/app/javascript/packs/hbw/components/form/submit.js.jsx

```diff
@@ -34,7 +34,9 @@ modulejs.define('HBWFormSubmit', ['React'], (React) => {
                 className={buttonCN}
                 disabled={formSubmitting || this.state.error}>
           <i className="fas fa-check" />
-          {` ${env.translator('submit')}`}
+          {submitButtonName === ''
```

Comment:
Don't compare this with empty string, this variable should be `null` if according form key is not set.